src/main/java/uk/gov/pay/webhooks/eventtype/EventTypeName.java

```diff
@@ -1,8 +1,11 @@
 package uk.gov.pay.webhooks.eventtype;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 import java.util.stream.Stream;
 
 public enum EventTypeName {
+    @JsonProperty("card_payment_captured")
     CARD_PAYMENT_CAPTURED("card_payment_captured");
```

Comment:
Relating to the validation on this PR, it feels a bit fiddly to have an enum with values that are completely unrelated to the underlying database values 
https://github.com/alphagov/pay-webhooks/blob/main/src/main/resources/migrations/0002_create_table_event_types.sql#L10

As the validation will only happen occasionally would it be reasonable to just lookup possible values from the database instead? (These might then also be used to associate the filtered list with the webhook entity)

This might be more complex if the enum is relied on by other parts of the code too.

Reply:
Hmm maybe. I quite like the fact that this enum (which is used all the way down from resource to dao) represents the fact that this can only have a limited set of values. e.g. I actually had to delete a test when I introduced it because I could no longer create a certain failure condition. But I agree that the disconnect with the source of truth is a bit annoying. I guess the true way is to have it as an enum in db, although that still wouldn't guarantee sync. When you say the validation will happen occasionally you mean 'every time a webhook is created'. Which isn't 1000s of tx per sec, but is not (or may not be) negligible either. Worth thinking about further